build/Dockerfile

```diff
@@ -16,20 +16,22 @@ RUN go env
 
 RUN CGO_ENABLED=0 go build -o /output/chaos-operator -v ./cmd/manager/
 
+FROM golang:alpine as cert
+RUN apk --no-cache add ca-certificates
+
 # Packaging stage
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
+FROM scratch
```

Comment:
Can we look into google's distroless image? I am thinking to update other control plane components.

cc: @ksatchit @uditgaurav